### DISCUSSION TITLE
Welcome to signserver-ce Discussions!

### DISCUSSION THREAD

**Discussion by fiddlermikey** [Keyfactor/signserver-ce#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by netmackan** [comment]
I'm Markus :wave: , been coding SignServer since 2009 and most of the time also as its product owner. 

Now as product architect at Keyfactor, I am still working on SignServer everyday and hope to be able to help out here and receive lots of ideas and contributions.

Feel free to ask or suggest anything and me or someone else in the community will try to answer. :rocket:

**Reply by ditalsastri8** [reply]
Halo, im Dita from Indonesian
i have error installation signserver, when install maven succes but, when next step to mvn help:effective-settings.
can u help me please.. it is my work and urgent

**Reply by netmackan** [reply]
Hi Dita,

Note, that normally you would not need to build SignServer from sources. If you unzip the the package called signserver-ce-x.y.z-bin.zip (ending in -bin.zip, see [Download and Unpack SignServer](https://doc.primekey.com/signserver/signserver-installation/install-signserver#InstallSignServer-DownloadandUnpackSignServer)) it is already built and ready to deploy.

If you still believe you need to build from sources, then please share what error message you got.


Cheers,
Markus

**Comment by mlundblad** [comment]
Hi!

I'm Marcus. Has been coding on SignServer since 2011. Now as a senior software engineer at Keyfactor. Coding daily on SignServer.

Feel free to ask or come with suggestions to me, and me or anyone else in the community will try to answer!

**Comment by ditalsastri8** [comment]
Hi

i want to tell if i can successfully install signserver using docker on
ubuntu.
next, i want to ask how to do setup for worker to be able to do
functionality from signserver?,
what API is suitable for signserver? an example that will be used from the
web that I will make later
and for CA i think we have to buy authentication acces, so can do signing
and verify?

thanks before


thanks

Pada tanggal Min, 14 Agu 2022 pukul 10.47 Markus Kilås <
***@***.***> menulis:
